internal/store/postgres/appeal_repository.go

```diff
@@ -231,17 +232,17 @@ func applyAppealFilter(db *gorm.DB, filters *domain.ListAppealsFilter) (*gorm.DB
 	}
 
 	if filters.CreatedBy != "" {
-		db = db.Where(`"appeals"."created_by" = ?`, filters.CreatedBy)
+		db = db.Where(`LOWER("appeals"."created_by") = LOWER(?)`, filters.CreatedBy)
```

Comment:
yes it's only possible to lowercase `"appeals"."created_by"` in SQL, but let's lowercase this value `filters.CreatedBy` at the application level instead of SQL?